index.html

```diff
@@ -601,6 +601,402 @@ <h6 class="float-left">START YOUR DAY WITH OUR TOP 5 ARTICLES</h6>
+                </div>
+                <div class="container-fluid px-3 py-3 col-md-6 col-lg-3">
+                    <div class="position-relative">
+                    <a href="#"><img class="w-100" src="./img/editor-pic.webp" alt="Editor picking"></a>
+                    <span class="border-bottom border-1 tag list-group-item text-danger bg-light fw-bold position-absolute start-0 bottom-0">U.S.</span>
+                    </div>
+                    <h3><a href="#" class="text-decoration-none text-dark">Gavin Newsom, Facing Recall, Accelerates Sluggish Vaccine Rollout</a></h3>
+                    <p>Slightly more than 1.3 million people in California had received a COVID-19 vaccine by Saturday.</p>     
+                </div>
+                <div class="container-fluid px-3 py-3 col-md-6 col-lg-3">
+                    <div class="position-relative">
+                    <a href="#"><img class="w-100" src="./img/editor-pic.webp" alt="Editor picking"></a>
+                    <span class="border-bottom border-1 tag list-group-item text-danger bg-light fw-bold position-absolute start-0 bottom-0">U.S.</span>
+                    </div>
+                    <h3><a href="#" class="text-decoration-none text-dark">Gavin Newsom, Facing Recall, Accelerates Sluggish Vaccine Rollout</a></h3>
+                    <p>Slightly more than 1.3 million people in California had received a COVID-19 vaccine by Saturday.</p>     
+                </div>
+                <div class="container-fluid px-3 py-3 col-md-6 col-lg-3">
+                    <div class="position-relative">
+                    <a href="#"><img class="w-100" src="./img/editor-pic.webp" alt="Editor picking"></a>
+                    <span class="border-bottom border-1 tag list-group-item text-danger bg-light fw-bold position-absolute start-0 bottom-0">U.S.</span>
+                    </div>
+                    <h3><a href="#" class="text-decoration-none text-dark">Gavin Newsom, Facing Recall, Accelerates Sluggish Vaccine Rollout</a></h3>
+                    <p>Slightly more than 1.3 million people in California had received a COVID-19 vaccine by Saturday.</p>     
+                </div>
+                
+            </div>
+        </section>
+        <section id="US">
+            <div class="caption d-flex align-items-center gap-2 mt-3 justify-content-start text-center pt-3 mx-3 border-top">                
+                <strong class="fs-4 text-danger">U.S.</strong>            
+            </div>
+            <div class="d-flex flex-column flex-md-row">
+                <div class="container-fluid px-3 py-3 col-md-4">                    
+                    <a href="#"><img class="w-100" src="./img/us-pic.webp" alt="crazy dude"></a>                    
+                    <strong><a href="#" class="text-decoration-none text-muted">Indiana Homeless Man Chooses to Be Arrested to Avoid Cold Weather</a></strong>
```

Comment:
News titles should have <hx> tags, so search engines can know of their importance and semantic word are used correctly. Also please make the titles black in color.